test-utils/src/lib.rs

```diff
@@ -16,14 +22,42 @@ use rand::Rng;
 use rand::seq::SliceRandom;
 use tezos_smart_rollup_utils::inbox::InboxBuilder;
 
-pub fn make_stepper_factory<MC: MemoryConfig, BCC: BlockCacheConfig>()
--> impl Fn() -> PvmStepper<NoHooks, MC, BCC> {
-    let program = fs::read("../../../assets/jstz").unwrap();
+pub struct TestConfig {
+    pub golden_dir: &'static str,
+    pub kernel_path: &'static str,
+    pub inbox_path: &'static str,
+}
+
+pub const DUMMY: TestConfig = TestConfig {
+    golden_dir: "tests/expected/dummy",
+    kernel_path: "../../../assets/riscv-dummy.elf",
+    inbox_path: "../../../assets/dummy-kernel-inbox.json",
```

Comment:
I think these paths will be a little awkward given they depend on where the test suit is run.

You can try a Cargo config trick:

```
[env]
OCTEZ_RISCV_ASSETS_DIR = { value = "assets", relative = true }
OCTEZ_RISCV_LIB_TESTS_DIR = { value = "src/riscv/lib/tests", relative = true }
```

You can retrieve the correct paths during compile-time with `env!("OCTEZ_RISCV_ASSETS_DIR")`.

Reply:
Done [here](https://github.com/tezos/riscv-pvm/compare/81b382dfb55c2052de145ddd0804d40652ab66a1..fb90774e614d282ec94dae035da7968a44fe66ce).